data/influxdb_urls.yml

```diff
@@ -12,18 +12,22 @@ cloud:
     - name: Amazon Web Services
       short_name: AWS
       regions:
-        - name: US West (Oregon)
```

Comment:
US West (Oregon) is the actual name of the AWS region (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-sample-templates-us-west-2.html) and is what we display in the UI. Same goes for all of the region names in this yml file.